lambdas/src/apis/content-v2/controllers/translator.ts

```diff
@@ -181,6 +181,7 @@ export async function getContents(fetcher: SmartContentServerFetcher, req: Reque
       // Let's try on the old content server
       const legacyContentServerResponse = await fetch(`https://content.decentraland.org/contents/${cid}`)
       if (legacyContentServerResponse.ok) {
+        LOGGER.info(`Failed to find '${cid}' on the content server, but found it on the legacy one.`)
```

Comment:
Should this be a warning?

Reply:
It's doing what it's supposed to be doing, I wouldn't consider the situation something that requires a warning